net/netavark/Makefile

```diff
@@ -17,8 +16,8 @@ PKG_BUILD_DEPENDS:= \
 	rust/host \
 	protobuf/host
 
-include $(INCLUDE_DIR)/package.mk
 include ../../lang/rust/rust-package.mk
+include $(INCLUDE_DIR)/package.mk
```

Comment:
why? `rust-package.mk` should be included after `package.mk`

Reply:
no idea how that happened, sorry about that :)